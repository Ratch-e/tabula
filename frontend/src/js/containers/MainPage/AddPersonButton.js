/**
 * Кнопка создания нового пользователя
 */
import * as React from "react";

class AddPersonButton extends React.Component {
  render() {
    return (
        <a className="button" href="/new_profile">Добавить анкету</a>
    );
  }
}

export default AddPersonButton;