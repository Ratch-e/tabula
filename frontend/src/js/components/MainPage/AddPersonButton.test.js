import React from 'react';
import { shallow } from 'enzyme';
import AddPersonButton from './AddPersonButton';

describe('Кнопка добавления пользователя', () => {
  it('рендерится корректно', () => {
    const component = shallow(<AddPersonButton />);
    expect(component.length).toMatchSnapshot();
  });

  it('переходит на нужную страницу', () => {
    const component = shallow(<AddPersonButton />);
    expect(component.find('.button').prop('href')).toBe('/new_profile');
  });
});
