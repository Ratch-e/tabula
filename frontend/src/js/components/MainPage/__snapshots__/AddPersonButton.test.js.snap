// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Кнопка добавления пользователя рендерится корректно 1`] = `1`;
